packages/cli/src/generator.test.ts

```diff
@@ -519,6 +524,27 @@ describe('getNamedImportNodes', () => {
 });
 
 describe('getNonTypeImports', () => {
```

Comment:
Perhaps a test for a type-only import would be useful? e.g. something like this:

```typescript
files: {
  '/index.ts': '// no-op',
  '/foo.ts': `
    export type Foo = number;
    export const bar: string = 'bar';
  `,
  '/bar.ts': `
    import { Foo } from './foo';
    export type Baz = Foo;
  `,
},
```

Presumably this would get correctly detected as a type-import and excluded from the output. This was the case we encountered in #76.

Reply:
Good point. I will add a test case for this!